src/trade-params/TradeParams.js

```diff
@@ -182,6 +196,11 @@ export default class TradeParams extends Component {
         this.updateTradeParams(updatedAmountPerPoint);
     }
 
+    onCloseModal = () => {
+        const { actions, index } = this.props;
```

Comment:
Intending to move away from modals as possible.
Just FYI.
And possibly all errors will be shown in the form.
We can also try showing all errors, not just the first one.